runtime/Makefile

```diff
@@ -11,7 +11,6 @@ CODE_COVERAGE?=OFF
 BUILD_TYPE?=RelWithDebInfo
 ENABLE_WARNINGS?=ON
 ENABLE_LIGHTNING?=ON
-ENABLE_LIGHTNING_KOKKOS?=ON
```

Comment:
Should the lightning git tag be updated to the one where catalyst LGPU is merged?

Reply:
It shouldn't be required as we ship the lightning_gpu standalone shared library with LGPU wheels. right?

Reply:
I'm not too familiar with how the lightning wheels are shipped, but in my experience when I made the seeding change to kokkos(' Catalyst interface) in lightning, I needed to update this git tag. 

I could also just be entirely wrong on this since I'm not super familiar with the wheels side of things. If that's the case feel free to ignore this comment ; )

Reply:
Discussed, this gittag update is unnecessary